src/Features/CSharp/Portable/SignatureHelp/AbstractCSharpSignatureHelpProvider.cs

```diff
@@ -62,14 +62,12 @@ protected static SymbolDisplayPart NewLine()
                 parameter.ToMinimalDisplayParts(semanticModel, position));
         }
 
+        /// <summary>
+        /// We no longer show awaitable usage text in SignatureHelp, but IntelliCode expects this
+        /// method to exist.
```

Comment:
Can we mark this as [Obsolete] now?

Reply:
(Look at other Obsolete attributes we have in the repo, we have a pattern for "this is a backcompat shim for <other thing>", which makes it easy to grep for things we can clean up.)